docs/tutorials/segmentation/annotate_segmentation.md

```diff
@@ -100,16 +100,16 @@ def circularity(perimeter, area):
 image = data.coins()[50:-50, 50:-50]
 label_image = segment(image)
 
-# create the properties dictionary
-properties = regionprops_table(
+# create the features dictionary
```

Comment:
The code changes come from napari's `examples/annotate_segmentation_with_text.py` example and should be consistent now.

Other changes in this file are mostly word swaps.